store/postgres/src/relational/prune.rs

```diff
@@ -170,14 +197,16 @@ impl TablePair {
                 // The conditions on `block_range` are expressed redundantly
                 // to make more indexes useable
                 sql_query(format!(
-                    "insert into {dst}({column_list}) \
+                    "/* controller=prune,phase=nonfinal,start_vid={next_vid},next_vid={batch_size} */ \
```

Comment:
Here and in a previous query there is `next_vid=batch_size`, which doesn't make sense to me.